packages/wallet/src/wallet.ts

```diff
@@ -159,7 +166,8 @@ export class Wallet {
             addressCount: this.addresses.length,
             createdAt: this.createdAt,
             network: this.network,
-            name: this.name
+            name: this.name,
+            emoji: this.emoji
```

Comment:
We can't store emoji in Wallet. We don't have such a field in Final JSON.

We only can make it readonly, based on UUID